staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -322,24 +291,12 @@ func (s *APIAggregator) AddAPIService(apiService *apiregistration.APIService) {
 	s.GenericAPIServer.Handler.NonGoRestfulMux.Handle(groupPath, groupDiscoveryHandler)
 	s.GenericAPIServer.Handler.NonGoRestfulMux.UnlistedHandle(groupPath+"/", groupDiscoveryHandler)
 	s.handledGroups.Insert(apiService.Spec.Group)
-}
-
-func (s *APIAggregator) deferLoadAPISpec(name string) {
-	s.specMutex.Lock()
-	defer s.specMutex.Unlock()
-	s.toLoadAPISpec[name] = 0
-}
-
-func (s *APIAggregator) deleteApiSpec(name string) {
-	s.specMutex.Lock()
-	defer s.specMutex.Unlock()
-	delete(s.apiServiceSpecs, name)
-	delete(s.toLoadAPISpec, name)
+	return nil
 }
 
 // RemoveAPIService removes the APIService from being handled.  It is not thread-safe, so only call it on one thread at a time please.
 // It's a slow moving API, so its ok to run the controller on a single thread.
-func (s *APIAggregator) RemoveAPIService(apiServiceName string) {
+func (s *APIAggregator) RemoveAPIService(apiServiceName string) error {
```

Comment:
I just added this to be consistent with `AddAPIService`. But it make sense. I removed this.